students/hyska_monika/lesson_07_manipulating_strings_datetime_formatting/functions_lesson07.py

```diff
@@ -0,0 +1,30 @@
+# function return length for the longest strings in array
+def find_max_sting(my_dict):
+    # sum set all strings from array to list, next search
+    # longest word(max), and return strings length
+    max_table = len(max(sum(my_dict, []), key=len))
+    return max_table
+
+
+# function return length for the longest string in array
+def find_max_sting2(my_dict):
+    max_str = len(my_dict[0][0])
+    for row in my_dict:
+        max_row = len(max(row, key=len))
```

Comment:
This is a maximum length of a cell in a row and in the end the function still returns one number instead of a list to use with the `rjust`. This is still not what is meant. You need a maximum cell length in each column, and then in each row for each cell you use max_column_length for this particular column.
```
max_column_length = [max([len(word) for word in column]) for column in zip(*table_data)]
...
        print(' '.join([cell.rjust(max_column_length[i]) for i, cell in enumerate(row)]))
```